Automattic/wp-calypso
# Help Center: Add option to suggest tailored posts

Related to #88707
Needs D142955-code

## Proposed Changes

* This PR adds a new feature on the Help Center, that allows “Tailored Articles” based on the URL that users are in. Basically, it is a mapping between URL's, posts and Blog Ids https://github.com/Automattic/wp-calypso/pull/88808/files#diff-1387095f7173d264c294187f77c20df1f285bad3ad7262e615718598da23ed65R2 .This mapping structure needs to be defined.

**PS** @donalirl is working on the URL structure that will be used in prod. Meanwhile, we are faking it.


## Testing Instructions



Before testing, make sure to test it on Simple sites and Atomic sites



Simple Sites

* Pull and run this branch
* This file (https://github.com/Automattic/wp-calypso/pull/88808/files#diff-1387095f7173d264c294187f77c20df1f285bad3ad7262e615718598da23ed65R2) maps which tailored article should be suggested per URL. I would advise getting your own articles from /support or /dev site.
* Navigating to a specific area of WPCom should map the file with the articles.



Atomic Sites

* Update this file https://github.com/Automattic/wp-calypso/pull/88808/files#diff-1387095f7173d264c294187f77c20df1f285bad3ad7262e615718598da23ed65R2 like with your testing articles
* Create an ETK new zip file from `apps/editing-toolkit/editing-toolkit-plugin` and upload it as a plugin to your atomic site.
* Navigating to a specific area of WPCom should map the file with the articles.

**Make sure to test also Help Center inside the editor `/post-new.php`**

## Pre-merge Checklist
